### PR TITLE
feat: pnpm import support yarn2 transform

### PR DESCRIPTION
**Pull request by fireairforce**
closes: https://github.com/pnpm/pnpm/issues/3993

### CODE REVIEW

**Comment by fireairforce** [comment]
I need to wait for the pr merged: https://github.com/snyk/nodejs-lockfile-parser/pull/131...

**Comment by ylemkimon** [comment]
Note Yarn 2+ lockfile is a valid YAML and can be parsed with any YAML parser.

**Comment by fireairforce** [comment]
Of course,So maybe we can implement by ourselves

File: pnpm-lock.yaml
Comment (zkochan) [inline pnpm-lock.yaml]: this looks like not correct path to the tarball. What registry do you use locally?

Comment (fireairforce) [reply pnpm-lock.yaml]: I have updated the lock file, sorry i just use the company registry...

File: packages/plugin-commands-installation/src/import.ts
Comment (zkochan) [inline packages/plugin-commands-installation/src/import.ts]: Don't put _ERR in the error code because each error code is automatically prepended with `ERR_PNPM_`

File: packages/plugin-commands-installation/src/yarnUtil.ts
Comment (zkochan) [inline packages/plugin-commands-installation/src/yarnUtil.ts]: maybe let's make a subdirectory for the import command and put this file there.

and add yourself as codeowner of it to `.github/CODEOWNERS`

```
/packages/plugin-commands-installation/src/import/ @fireairforce 
```

Comment (fireairforce) [reply packages/plugin-commands-installation/src/yarnUtil.ts]: ok, thx, i will do this

**Comment by fireairforce** [comment]
image

the ci failed but it works fine at my local env..

**Comment by zkochan** [comment]
I'll check later.

**Comment by fireairforce** [comment]
I have updated the case, it will run successfully.

**Review by zkochan — APPROVED** [review]

**Pull request merged**
